test/nodetool/test_snapshot.py

```diff
@@ -99,3 +102,195 @@ def test_listsnapshots(nodetool, request):
         assert res == scylla_expected_output
     else:
         assert res == cassandra_expected_output
+
+
+def check_snapshot_out(res, tag, ktlist, skip_flush):
+    if type(tag) is int:
+        tag_pattern = str(tag)[:-5] + "[0-9]{5}"
```

Comment:
I thought it is implied, I omit `nodetool` from names because we are in `test/nodetool` directory here. :)

Reply:
This is a utility function that does something *inside* this file. It checks the standard output of the "nodetool snapshot" command - this can either be explained in a comment or a good name. As it stands, the name "check_snapshot_out" led me to misunderstand it checks the actual snapshot. Which isn't even possible because everything in this test is mocked and no actual snapshot is ever created.
Anyway, it was just a thought. You don't have to change it.

Reply:
Added comment for the method in v2. Also, this tricky tag matching is much clearer in v2.